rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml

```diff
@@ -12,20 +12,25 @@ injection.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License"
 name = "Suspicious Endpoint Security Parent Process"
 risk_score = 47
 rule_id = "b41a13c6-ba45-4bab-a534-df53d0cfed6a"
 severity = "medium"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 timeline_id = "76e52245-7519-4251-91ab-262fb1a1728c"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
- process.name:(esensor.exe or "elastic-endpoint.exe" or "elastic-agent.exe") and
- not process.parent.executable:"C:\Windows\System32\services.exe"
+process where event.type in ("start", "process_started", "info") and
+ process.name in ("esensor.exe", "elastic-endpoint.exe", "elastic-agent.exe") and
```

Comment:
I think this will FP on the installer for `elastic-agent`. You should be safe to remove `elastic-agent.exe`

Reply:
yep make sense